### PR TITLE
Add some metadata about Bulgaria and Greece

### PR DESCRIPTION
**Pull request by Dimitar5555**

### CODE REVIEW

File: res/country_metadata/popularReligions.yml [hunk 1]
```diff
@@ -7,6 +7,7 @@ AU: [christian, muslim, buddhist]
 AZ: [muslim]
 BD: [muslim, hindu]
 BF: [muslim]
+BG: [christian, muslim]
```
Comment (matkoniecz) [inline res/country_metadata/popularReligions.yml:10]: Note that it is default already, maybe it should be mentioned somehow in this file?

Either have `default: [christian, muslim, buddhist, hindu, jewish]` or `# note that adding order here is needed only if default order from AddReligionForm needs to be overriden`?

Comment (Dimitar5555) [reply res/country_metadata/popularReligions.yml:10]: It says that the default is `[]` https://github.com/Dimitar5555/StreetComplete/blob/patch-1/res/country_metadata/popularReligions.yml#L3

Comment (matkoniecz) [reply res/country_metadata/popularReligions.yml:10]: Yes, but it is used solely to change order in https://github.com/streetcomplete/StreetComplete/blob/0857679a146abaf474f051f8e8d12225d9a10ff3/app/src/main/java/de/westnordost/streetcomplete/quests/religion/AddReligionForm.kt#L19 (so default is actually `[christian, muslim, buddhist, hindu, jewish...`)

Comment (Dimitar5555) [reply res/country_metadata/popularReligions.yml:10]: Is it possible to just set the default to `[christian, muslim, buddhist, hindu, jewish...]` in `popularRegligions.yml` or would this break something?

Comment (westnordost) [reply res/country_metadata/popularReligions.yml:10]: it would be possible. I don't think it would break anything.

Comment (matkoniecz) [reply res/country_metadata/popularReligions.yml:10]: It actually breaks something.

screen03

Comment (westnordost) [reply res/country_metadata/popularReligions.yml:10]: Hm but why??

Comment (matkoniecz) [reply res/country_metadata/popularReligions.yml:10]: I found and fixed problem in https://github.com/streetcomplete/StreetComplete/commit/e8c9b295d598991040ff59dc9044b6f48317a9f0 (in way nicer than my initial attempt but maybe still can be refactored)

The problem was that -1 (not found) was sorted before things at 0th, 1st, 2nd.. positions.

**Comment by matkoniecz** [comment]
OK, religion issues are now fixed.

Ordering will now work properly and config file should be less confusing, default is now longer empty.

I applied your commits about road signs - and welcome on https://github.com/streetcomplete/StreetComplete/graphs/contributors

Thanks for this contribution and you are welcome to contribute more if you want!

If you are looking for something relatively simple or without significant programing feel free to ask.

This PR will be marked as closed but changes are applied (done this way as street signs were applied with https://github.com/streetcomplete/StreetComplete/commit/5422234cd17f220101866c636c9c07bcbabc0ee5 - author info preserved and religion part was replaced by changing default)

**Pull request closed without merging**